src/views/Timesheets/index.jsx

```diff
@@ -613,7 +613,7 @@ const Notice = () => (
   </Container>
 );
 
-const switchOverDate = new Date('2023-08-27 00:00');
+const switchOverDate = new Date('2023-08-22 12:00');
```

Comment:
This should probably come from the database in the long term, right?

Reply:
Well no, in the long term we can just delete all the Timesheets code. I was just leaving it in for now in case we need to quickly put timesheets back up.